src/main.cpp

```diff
@@ -4364,18 +4370,19 @@ std::string GetWarnings(const std::string& strFor)
     // Misc warnings like out of disk space and clock is wrong
     if (strMiscWarning != "")
     {
-        strStatusBar = strGUI = strMiscWarning;
+        strStatusBar = strMiscWarning;
+        strGUI += strGUI.empty() ? "" : uiAlertSeperator + strMiscWarning;
```

Comment:
Nit: That was confusing to read. Mind to add curly brackets around the ternary operator?